index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Animal Trading Cards</title>
+	<link rel="stylesheet" type="text/css" href="./styles.css" />
+</head>
```

Comment:
add in a `viewport` metatag so your site scales better on mobile.
`<meta name="viewport" content="width=device-width, initial-scale=1">`
https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag